pkg/cmd/checkendpoints/controller/connection_checker.go

```diff
@@ -186,7 +185,7 @@ func isDNSError(err error) bool {
 
 // manageStatusLogs returns a status update function that updates the PodNetworkConnectivityCheck.Status's
 // Successes/Failures logs reflect the results of the check.
-func manageStatusLogs(check *operatorcontrolplanev1alpha1.PodNetworkConnectivityCheck, checkErr error, latency *trace.LatencyInfo) []v1alpha1helpers.UpdateStatusFunc {
+func manageStatusLogs(check *operatorcontrolplanev1alpha1.PodNetworkConnectivityCheck, checkErr error, latency *trace.LatencyInfo) ([]v1alpha1helpers.UpdateStatusFunc, time.Time) {
```

Comment:
update doc. This time is the time the check started?

Reply:
Updated.